src/streaming/StreamProcessor.js

```diff
@@ -114,18 +113,14 @@ function StreamProcessor(config) {
             requestModifier: RequestModifier(context).getInstance()
         });
 
-        representationController = RepresentationController(context).create();
-        representationController.initialize(this);
-
         fragmentModel = scheduleController.getFragmentModel();
         fragmentModel.setLoader(fragmentLoader);
+
+        representationController = RepresentationController(context).create();
+        representationController.initialize(this);
     }
 
     function reset(errored) {
-        if (fragmentModel) {
-            fragmentModel.reset();
-            fragmentModel = null;
-        }
```

Comment:
It is cleaned up in FragmentController and persisted over new creations of schedule controller.  It is using the same one if it exists in the init of schedule controller.